src/components/ui/TabButtonGroup/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { View } from "react-native";
+import { Button } from "react-native-elements";
+import { RFValue } from "react-native-responsive-fontsize";
+import styles from "./styles";
+import { colors } from "@app/utils";
+
+interface TabButtonGroupProps {
+  id?: string;
+  onPress: (index: number) => void;
+  selectedIndex: number;
+  buttons: string[];
+}
+
+const TabButtonGroup: React.FC<TabButtonGroupProps> = ({
+  id,
```

Comment:
don't hate us but we started destructuring the props...

Reply:
ok

Reply:
is it ok

Reply:
maybe we should have a meeting about conventions